util/openocd/target/lowrisc-earlgrey-silicon.cfg

```diff
@@ -0,0 +1,9 @@
+# Copyright lowRISC contributors.
```

Comment:
I might rename this file to `nuvoton-earlgrey-z1-silicon.cfg` wdyt?

Reply:
I'll rename and push the change.